2021-04-05 20:27:29,808-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:19:27,687-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:19:57,478-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:26:00,473-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:26:07,967-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:30:17,730-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:31:19,807-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:32:55,545-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:33:43,511-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:43:27,081-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:45:57,139-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:49:06,029-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:49:06,198-DEBUG-FUNC:main-LINE:57: Running main function
2021-04-05 21:50:50,217-DEBUG-FUNC:<module>-LINE:8: Connecting to the database with pymysql
2021-04-05 21:50:50,228-DEBUG-FUNC:main-LINE:57: Running main function
